dhall/src/Dhall/Util.hs

```diff
@@ -122,13 +125,21 @@ data Censor = NoCensor | Censor
 -- | Path to input
 data Input = StandardInput | InputFile FilePath
 
+-- | Path to input or raw input text, necessary since we can't read STDIN twice
+data InputOrTextFromStdin = PathTo Input | StdinText Text
```

Comment:
`PathTo StandardInput` seems like a bit of a strange combination to me. How about
```suggestion
data InputOrTextFromStdin = Input_ Input | StdinText Text
```

Reply:
Done.